### ISSUE TITLE
"In the News" commentaries missing search form label

### ISSUE THREAD

**Issue by erinchambers** [CDRH/african_poetics#334]
https://africanpoetics.unl.edu/inthenews/commentaries/search

"Search for..." is missing a form label
`<input type="text" name="q" id="q" placeholder="Search for..." class="form-control" style="">`

May need to check other search pages

**Comment by erinchambers** [comment]
Will fixed this but I made a slight tweak so that the label is hidden except for screen readers; it is duplicated by the "Search Poets" heading already on the page
